drizzlepac/outputimage.py

```diff
@@ -729,8 +730,9 @@ def addWCSKeywords(wcs, hdr, blot=False, single=False, after=None):
 
     # Update WCS Keywords based on PyDrizzle product's value
     # since 'drizzle' itself doesn't update that keyword.
-    hdr['WCSNAME'] = wname
-    hdr['WCSTYPE'] = wtype
+    if wname != '':
```

Comment:
Please add a comment here indicating this is a temporary fix until the WCSNAME issue is resolved for MVM.

Reply:
Additional comments were added in 3b36622.  It is not clear that this specific change is temporary, so the comments simply clarified the logic that was implemented.  The definition of the MVM WCSNAME value will not require this logic to be removed, since this logic causes the WCSNAME to revert to what was used to align the input exposures as opposed to the MVM WCS definition which is not associated with any catalog or alignment fit.